src/index.ts

```diff
@@ -35,15 +35,14 @@ export interface Options {
 
 export function process(content: string, filename: string, config: any) {
   const options: Options = getOptions(config)
+  const enableSourcemaps =  typeof options?.sourcemap == 'undefined' || options?.sourcemap
 
   const ext = getExt(filename)
   const loader = options?.loaders && options?.loaders[ext] 
     ? options.loaders[ext]
     : extname(filename).slice(1) as Loader
 
-  const sourcemaps = options?.sourcemap 
-    ? { sourcemap: true, sourcefile: filename }
-    : {}
+  const sourcemaps: Partial<TransformOptions> = enableSourcemaps ? { sourcemap: "external", sourcefile: filename } : {}
```

Comment:
@airhorns we can set the `sourcemap` to `true`, and `sourcesContent` to `false` so that it will not generate `sourcesContent` since at the end you are setting it to `null`

Reply:
Ah ha -- I didn't realize there was a `sourcesContent` option, good call!